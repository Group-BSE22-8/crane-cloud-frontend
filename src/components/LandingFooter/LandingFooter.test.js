import React from "react";

import { shallow } from 'enzyme';

import LandingFooter from './index'

import NewLogo from "../NewLogo";

import { ReactComponent as TwitterIcon } from "../../assets/images/twitter.svg";

import { ReactComponent as FacebookIcon } from "../../assets/images/facebook.svg";

import { ReactComponent as LinkedinIcon } from "../../assets/images/Linkedin.svg";

import { ReactComponent as MediumIcon } from "../../assets/images/medium.svg";

describe ('Test the LandingFooter component', () => {
    it('checks if landing footer component rendered correctly', () => {
        const FooterComponent = shallow(<LandingFooter />);
        
        expect(FooterComponent).toMatchSnapshot();
        
        expect(FooterComponent.contains(<NewLogo />)).toEqual(true);

        expect(FooterComponent.contains(<TwitterIcon />)).toEqual(true);

        expect(FooterComponent.contains(<FacebookIcon />)).toEqual(true);

        expect(FooterComponent.contains(<LinkedinIcon />)).toEqual(true);

        expect(FooterComponent.contains(<MediumIcon />)).toEqual(true);
    })
})