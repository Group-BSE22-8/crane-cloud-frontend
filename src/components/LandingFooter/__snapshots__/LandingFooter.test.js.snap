// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the LandingFooter component checks if landing footer component rendered correctly 1`] = `ShallowWrapper {}`;
